help/de/Help_ch_Account-Actions.xml

```diff
@@ -902,20 +954,24 @@
             </para>
           </listitem>
         </orderedlist>
-
-        <para>Wenn Sie die Kurse per Kommandozeile abrufen wollen, können Sie dazu den folgenden Befehl benutzen:
-        </para>
-
-        <para><code>gnucash --add-price-quotes &lt;gnucash-file-name&gt;</code>
+<!-- FIXME source: new footnote! -->
+        <para>Wenn Sie die Kurse per Kommandozeile abrufen wollen, können Sie dazu den folgenden Befehl benutzen
+          <code language="shell">gnucash-cli --quotes get &lt;gnucash-file-name&gt;</code> und die
+          Kurs direkt in ihre &app;-Datendatei schreiben.
```

Comment:
Kurse **werden** direkt in ihre &app;-Datendatei **geschrieben**.

Reply:
ok.